src/extensions/hive/components/PublishInfo.js

```diff
@@ -0,0 +1,100 @@
+import React from 'react';
```

Comment:
Can we change this `.js` file to `.ts` so that we only have typescript files in the src folder?

Reply:
yes

Reply:
changed extension to tsx